delivery.yaml

```diff
@@ -8,8 +8,6 @@ pipeline:
   commands:
   - desc: build-push
     cmd: |
-      apt-get update  -o Acquire::http::AllowRedirect=false
-      apt-get install -o Acquire::http::AllowRedirect=false -y bzr redis-server jq
```

Comment:
I don't see `bzr` used anywhere but I think this would break a Makefile that depends on `jq`